src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import java.util.List;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+import duke.task.Task;
+
```

Comment:
This class is missing a JavaDoc header. Its constructor is also missing JavaDoc comment.